modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -81,13 +97,70 @@ export default function AuditBarChart({rtl, vocabularies}) {
 		? [BAR_CHART.dotRadiusMin]
 		: [BAR_CHART.legendHeight + BAR_CHART.dotRadiusMin];
 
+	const [checkboxes, setCheckbox] = useState(legendCheckboxes);
+
+	useEffect(() => {
+		if (Object.keys(colors).length) {
+			const head =
+				document.head || document.getElementsByTagName('head')[0];
```

Comment:
Why not just `document.head`?

[TypeScript thinks it is guaranteed to be non-null](https://github.com/microsoft/TypeScript/blob/cc7cb9eadb10a56687513d4fc3a67d5210b387bb/src/lib/dom.generated.d.ts#L4505), and [MDN doesn't raise any red flags](https://developer.mozilla.org/en-US/docs/Web/API/Document/head) in the browser compatibility section.

Reply:
Ok, I'll change it. Thanks!